src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -633,7 +642,8 @@ public static void joinGame(final GameDescription description, final Messengers
     }
     final Version engineVersionOfGameToJoin = new Version(description.getEngineVersion());
     String newClassPath = null;
-    if (!ClientContext.engineVersion().getVersion().equals(engineVersionOfGameToJoin)) {
+    final boolean areSameVersion = ClientContext.engineVersion().getVersion().equals(engineVersionOfGameToJoin);
```

Comment:
nit, `areSameVersion` is a really awkward name. `sameVersion` is shorter, and would read much better IMO